### ISSUE TITLE
Reloading configuration

### ISSUE THREAD

**Issue by Lillecarl** [zellij-org/zellij#1646]
Hi!

It's not uncommon that I keep a Zellij session running for days, maybe weeks. Sometimes in the middle of one of those sessions I see that my Keybindings doesn't cover all usecases I have (Since I'm using the Vim keybinding template).

Now editing the configuration to add the keybindings is no problem, but now there's no way for me to reload the keyconfig without restarting Zellij.

A small convenience feature that might be architecturally hard to implement, but I figured I'd leave it here as a suggestion. WezTerm (another rust project) even monitors the configuration file and reloads as soon as it receives an inotify.

**Comment by jaeheonji** [comment]
This seems to be a duplicate of #1405

**Comment by Lillecarl** [comment]
Indeed, i searched for "reload" and didn't find anything. Thanks @jaeheonji

**Comment by phyber** [comment]
I'm not sure this is really a duplicate, and it should possibly be reopened. The other issue is specifically asking for reloading of keybinds, while this issue it asking for reloading of the whole configuration file. I understand why it was closed given the context in which the question was asked, but the two really are individual features.

I'm configuring Zellij for the first time and reloading of the whole configuration file would be very useful. I'm playing with UI options, themes, layouts, etc, and having to restart for small changes isn't a great experience.

Even outside of that, I expect many users also have Zellij running for weeks or months, and perhaps in that time they'd like to reconfigure it without having to kill everything that's running.

It's possible that reloading of the keybindings alone is a step towards having the ability to reload the whole configuration file. So perhaps this feature request depends on #1405, rather than duplicating it?

**Comment by rpodgorny** [comment]
this is definately not a duplicate. can we reopen, please?

**Comment by boomshroom** [comment]
1405 seems like a subset of this. So this issue being fixed would automatically mean it's fixed, but it getting fixed still leaves elements of this issue undo.

**Comment by jaeheonji** [comment]
#3386
